### PR TITLE
chore: add Azure DevOps Pipelines documentation

### PR DESCRIPTION
**Pull request by notdryft**

### CODE REVIEW

File: content/reference/plugins/azure_devops/index.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+
+pool: AzurePipelinesPool # Name of the agent pool configured in Azure DevOps
+
+steps:
+- task: Bash@3
+  displayName: Install JQ
+  inputs:
+    targetType: 'inline'
+    # Curl and JQ are required by the CI script
+    script: 'sudo apt-get update && sudo apt-get -y install jq'
```
Comment (Isammoc) [inline content/reference/plugins/azure_devops/index.md:53]: The comment mention both curl and jq, but the script only install jq.

File: content/reference/plugins/azure_devops/index.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+On Azure DevOps, click on **Pipelines** in the left menu, then on **Create Pipeline** or **New Pipeline**.
+
+From here, you will follow a 4 steps process: Connect -> Select -> Configure -> Review.
+
+1. On the **Connect** step, select the type of the repository you want to use. We will be using GitHub.
+2. On the **Select** step, select the repository your created earlier.
+
+If you never connected Azure DevOps to your GitHub account, you'll be prompted by GitHub to authorize the Azure DevOps application to access your repositories on your behalf.
+
+After your account is linked, you will be able to browse your repositories. When using an organization repository, click on *All repositories* on the top right.
```
Comment (Isammoc) [inline content/reference/plugins/azure_devops/index.md:77]:
```suggestion
After your account is linked, you will be able to browse your repositories. When using an organization repository, click on **All repositories** on the top right.
```

File: content/reference/plugins/azure_devops/index.md [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+
+3. On the **Configure** step, as we already made a `azure-pipelines.yml` file, the step will be skipped and you will be asked to review its content directly.
+
+4. On the **Review** step, create three variables by clicking **New variable** on the top right, then:
+
+    - `gatlingEnterpriseUrl`: use `https://cloud.gatling.io`
+    - `apiToken`: make sure **Keep this value secret** is enabled
+    - `simulationId`: use the ID of the simulation you want to run
+
+Here is what it look like:
```
Comment (Isammoc) [inline content/reference/plugins/azure_devops/index.md:87]:
```suggestion
Here is what it looks like:
```

File: content/reference/plugins/azure_devops/index.md [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+
+<div style="margin: 0 auto; max-width: 350px">
+{{< img src="variables-configured.png" alt="Variables all configured" caption="Variables all configured" >}}
+</div>
+
+Click on **Save**, and then on **Run** to start the job, after which you will be redirected to the Job summary page.
```
Comment (Isammoc) [inline content/reference/plugins/azure_devops/index.md:93]:
```suggestion
Click on **Save**, and then on **Run** to start the job. You will be redirected to the Job summary page.
```

File: content/reference/plugins/azure_devops/index.md (hunk 1 above)
Comment (notdryft) [reply content/reference/plugins/azure_devops/index.md:53]: Add "In this example, we use an Ubuntu image which comes with curl but not jq"

**Review by Isammoc — APPROVED** [review]

**Pull request merged**